src/main/java/org/teamresistance/frc/subsystem/shoot/ShooterController.java

```diff
@@ -0,0 +1,49 @@
+package org.teamresistance.frc.subsystem.shoot;
+
+import edu.wpi.first.wpilibj.PIDSourceType;
+import org.strongback.control.SoftwarePIDController;
+import org.teamresistance.frc.Pose;
+import org.teamresistance.frc.subsystem.Controller;
+import org.teamresistance.frc.util.SynchronousPID;
+
+import static org.teamresistance.frc.IO.shooterEncoder;
+
+/**
+ * @author Tarik C. Brown
+ */
+public class ShooterController implements Controller {
+  private SynchronousPID shootPID;
+  private static final double TOLERANCE = 2;
+  private static final double KP = 0.0;
+  private static final double KD = 0.0;
+  private static final double KI = 0.0;
+  private static final double FF = 0.0;
+  double targetRate;
+
+  public ShooterController(SynchronousPID shootPID) {
+    this.shootPID = new SynchronousPID(SoftwarePIDController.SourceType.DISTANCE, KP, KI, KD, FF)
+        .withConfigurations(controller -> controller
+            .withOutputRange(-1.0, 1.0) // motor
```

Comment:
Don't forget to add a `.withInputRange(min, max)` when you figure out the input range of your encoder. :^)